raw/mock_exam/docs/questions/question-12.adoc

```diff
@@ -0,0 +1,15 @@
+// tag::DE[]
+=== Frage 12
```

Comment:
in principle, an empty questions could be removed. But: several people have forked this repo and created their own (technical) representations without the longer IDs..

For those I propose to keep the original numbering

Reply:
Didn't know that a GitHub repository is kind of an interface that isn't allowed to be changed because of forks 😉.

I understand your point, but it seems like a technical debt to me (which is unfortunate for such a young and yet to be released project).

Reply:
proposal: invent a new question, so that we can have both... a question 12 and no new numbering for existing questions :-)

(no, moving question 39 to this is not allowed..)

Reply:
Deal! (here is my try, but maybe to easy?)


Welche ZWEI Antworten sind falsch in Bezug auf Architekturentscheidungen? 

(y) Architekturentscheidungen müssen nie aufgeschrieben werden, da diese bereits dem Entwicklungsteam bekannt sind.
(n) Architecture Decision Records helfen dabei, Entscheidungen in ihrem Kontext nachzuvollziehen.
(y) Nachdem man sich für ein Framework entschieden hat, darf diese Entscheidung nicht mehr geändert werden.
(n) Qualitätsanforderungen helfen maßgeblich bei Architekturentscheidungen.

Which TWO answers are incorrect regarding architecture decisions? 

(y) Architecture decisions never need to be written down because they are already known to the development team.
(n) An architecture decision record helps to make the decision's context understood.
(y) Once a decision has been made on a framework, that decision must not be changed.
(n) Quality requirements help significantly with architecture decisions.

Reply:
hhm - will make it a K-question, so people have to select true/false for every option.